src/Microsoft.Graph.Core/Exceptions/ErrorDetails.cs

```diff
@@ -0,0 +1,85 @@
+﻿// ------------------------------------------------------------------------------
```

Comment:
File is called ErrorDetails but the class is ErrorDetail.  Can we rename the file match the class?

Reply:
addressed